models/game.rb

```diff
@@ -1,23 +1,29 @@
 class Game
 
-  def initialize(board)
+  attr_reader :players
+
+  def initialize(board, players)
     @board = board
+    @players = players
   end
 
   def get_grid
     @board.grid
   end
 
-  def play_game(move)
-    @board = @board.new_board(move, get_mark)
+  def play_turn(human_move)
+    play_human_move(human_move)
+    if current_player.name == "Computer Player"
```

Comment:
Having the players be inherited from Player and then running conditionals on their name feels weird. Is there anyway you can refactor the code so it doesn't matter if it's a human or a  player?

Reply:
If not, I'd make the names be constants so you aren't just using random strings